runelite-client/src/main/java/net/runelite/client/plugins/zoom/ZoomPlugin.java

```diff
@@ -100,7 +97,7 @@ public void onScriptCallbackEvent(ScriptCallbackEvent event)
 
 		if ("outerZoomLimit".equals(event.getEventName()))
 		{
-			int outerLimit = Math.max(OUTER_CONFIG_ZOOM_LIMIT_MIN, Math.min(OUTER_CONFIG_ZOOM_LIMIT_MAX, zoomConfig.outerLimit()));
+			int outerLimit = zoomConfig.outerLimit();
```

Comment:
The `@Range` is really supposed to be config-panel only enforced, you should not assume the value from the config is in a valid range probably

Reply:
Well as there is no good way to get the actual range, and the only way to make this wrong is from config, i think it is probably fine? Even though with config from older release this can be a problem

Reply:
The other change I made should solve the issue of old configs being out of range

Reply:
If you remove the resetting of the config, the other change is actually only +1 line which is `constrainToRange` on value. I would do that and then readd the min/max checks here (also constrainToRange actually).

Reply:
Thanks I didn't know about Ints.constrainToRange()

Forcing an update of the config was flawed anyway because that only runs when the config panel is opened. It would be possible for a user to carry over an invalid config for as long as they didn't open the config options.

Therefore I've re-added the min/max checks to ZoomPlugin.

Is the way I've done this okay? I exposed the min and max values in the config to guarantee that they're consistent between the config and the plugin.

Reply:
This looks much better.